<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accordion</class>
 <widget class="QWidget" name="Accordion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>780</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleVerifyCert {
border-top: 1px solid #E7EAEF;
border-bottom: 1px solid #E7EAEF;
}
#authBox, #signBox {
border-right: 1px solid #E7EAEF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="AccordionLayout" stretch="0,1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="AccordionTitle" name="titleVerifyCert">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>PIN/PUK codes and certificates</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentVerifyCert" native="true">
     <layout class="QHBoxLayout" name="contentVerifyCertLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="VerifyCert" name="authBox" native="true"/>
      </item>
      <item>
       <widget class="VerifyCert" name="signBox" native="true"/>
      </item>
      <item>
       <widget class="VerifyCert" name="pukBox" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="accordionSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccordionTitle</class>
   <extends>QCheckBox</extends>
   <header>widgets/AccordionTitle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VerifyCert</class>
   <extends>QWidget</extends>
   <header>widgets/VerifyCert.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
